src/nucname.h

```diff
@@ -138,7 +138,8 @@ namespace nucname
         NaNEstr += " --> ";
         NaNEstr += nucnow;
       }
-      return (const char *) NaNEstr.c_str();
+      const char* NaNEstr_rtn = NaNEstr.c_str();
```

Comment:
This may be obvious, but why are you keeping `const char* ` in the changes to this file (`nucname.h`) and not in the one for the above file (`h5wrap.h`). The changelog says these may cause some problems.

Reply:
this was not intended.
Let me push a fix...

thx for catching this